src/styles/blocks/calendar.scss

```
@@ -0,0 +1,40 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
+  width: calendar-width($day-size,$day-gap,$calendar-padding);
+  padding: 10px;
+
+  &__day {
+    @include day-square($day-size);
+    background-color: $day-color;
+    border: 1px solid black;
+    text-align: center;
+    line-height: 100px;
+    cursor: pointer;
+    transition: background-color 0.5s, transform 0.5s;
```

If you set same transition duration for all of the properties, you  don't really need to double it.
```suggestion
    transition: all 0.5s;
```

or

```suggestion
    transition-duration: 0.5s;
```